pkg/api/validation/schema.go

```diff
@@ -311,6 +311,9 @@ func (s *SwaggerSchema) validateField(value interface{}, fieldName, fieldType st
 		return s.ValidateObject(value, fieldName, fieldType)
 	}
 	allErrs := []error{}
+	if reflect.TypeOf(value) == nil {
```

Comment:
IsZero() || IsNil() would be correct then.

On Wed, May 18, 2016 at 4:04 AM, Michail Kargakis <notifications@github.com>
wrote:

> In pkg/api/validation/schema.go
> <https://github.com/kubernetes/kubernetes/pull/25709#discussion_r63661168>
> :
>
> > @@ -311,6 +311,9 @@ func (s *SwaggerSchema) validateField(value interface{}, fieldName, fieldType st
> >  		return s.ValidateObject(value, fieldName, fieldType)
> >  	}
> >  	allErrs := []error{}
> > +	if reflect.TypeOf(value) == nil {
>
> Hm, I would expect IsNil to work. Anyway. Can you also add a test in
> hack/test-cmd.sh that tests that the panic is fixed?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/25709/files/c6fcaa8d4bd3d37246626c236eef2df907eeb592#r63661168>
>